prow/plugins/verify-owners/verify-owners.go

```diff
@@ -334,6 +334,13 @@ func handle(ghc githubClient, gc git.ClientFactory, roc repoownersClient, log *l
 		}
 	}
 
+	var joinOrgURL string
+	if triggerConfig.JoinOrgURL != "" {
+		joinOrgURL = triggerConfig.JoinOrgURL
+	} else {
+		joinOrgURL = fmt.Sprintf("https://github.com/orgs/%s/people", org)
```

Comment:
I didn't look into this too closely, but possibly it would be possible to centralize this defaulting in `prow/plugins/config.go` or some such, instead of relying on consumers to remember to default locally.